src/language-js/print/array.js

```diff
@@ -9,17 +9,18 @@ import {
   fill,
 } from "../../document/builders.js";
 import hasNewline from "../../utils/has-newline.js";
+import isNextLineEmptyAfterIndex from "../../utils/is-next-line-empty.js";
+import { skipInlineComment, skipTrailingComment } from "../../utils/public.js";
```

Comment:
Missed this on previous review. Do not import from `public.js`, please import from the actual file directly.